docs/configuration/settings.rst

```diff
@@ -5,113 +5,469 @@ Mautic Configuration Settings
+  :width: 600
+  :alt: Screenshot showing Company Merge Settings Configuration in Mautic
+
+* **Merge by unique fields with operator** - When you use more than one unique field identifier, you can define how the search merges them.
+
 Notification settings
 *********************
 
-Campaign notification settings
-==============================
+.. image:: images/campaign-notification-settings.png
+  :width: 600
+  :alt: Screenshot showing Campaign Notification Settings Configuration in Mautic
+
+.. image:: images/webhook-notification-settings.png
+  :width: 600
+  :alt: Screenshot showing Webhook Notification Settings Configuration in Mautic
+
+If a Campaign or Webhook is automatically unpublished because of a high volume of errors, Mautic sends a notification alerting Users.
+
+* **Send notification to author** - Set this field to Yes to send an Email notification to the User who created the unpublished Campaign or Webhook. If the User is deleted, no notification is sent.
+
+* **Email addresses to receive notifications** - To send notifications to Users other than the author, enter a comma-separated list of Email addresses. You can include distribution lists and addresses of the Users who are not present in the Mautic instance.
 
-Webhook notification settings
-=============================
 
 Landing page settings
 *********************
 
+.. image:: images/landing-page-settings.png
+  :width: 600
+  :alt: Screenshot showing Landing Page Settings Configuration in Mautic
+
+* **Show Category in page URL?** - If you use Categories, the landing page’s associated Category displays in the URL if you select Yes.
+
+* **Analytics script** - To track landing page visits and activity in other platforms such as Google Analytics, add those tracking scripts here.
+
 Tracking settings
 *****************
 
 Mautic tracking settings
 ========================
 
+.. image:: images/tracking-settings.png
+  :width: 600
+  :alt: Screenshot showing Tracking Settings Configuration in Mautic
+
+* **Tracking code** - Insert this code on any page you would like to have tracked in Mautic before the ending </body> tag.
+
+.. note:: 
+
+    The default tracking code provided in a new instance updates and changes after you set up a new custom domain or when you make changes to an existing one. You must use the new tracking code that reflects the new or edited custom domain. If you are using the plugin for WordPress, Drupal, or Joomla, re-enter your account information in the plugin.
+
+* **Identify visitor by tracking URL** - Select **Yes** to have Mautic begin tracking a Contact after the Contact clicks a link in an Email on a device where no cookie exists.
+
+* **Anonymize IP** - Select **Yes** to not store full IP addresses for your visitors/Contacts. This setting aids customers in achieving GDPR compliance.
+
+* **Identify visitors by IP** - Select **Yes** to enable, unidentified visitors with the same IP address as an existing Contact will be tracked as that Contact.
```

Comment:
```suggestion
* **Identify visitors by IP** - Select **Yes** to use the IP address to identify Contacts. Unidentified visitors with the same IP address as an existing Contact will be tracked as that Contact. This may result in undesirable outcomes with large companies who use the same externally facing IP address.
```